### DISCUSSION TITLE
Timezone in OS and yeti

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16190] [How to]
Operating system time zone is different that set in yetiforce. Can it cause a problems?
Best

**Comment by mariuszkrzaczkowski** [comment]
no, the time zone is saved in the CRM configuration file.

**Comment by s2b-git** [comment]
Asking as for instance "add_time" built-in function in workflows relay on timezone set on os level... or am i wrong?
